src/components/PeopleList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../types/Person';
+
+interface Props {
+  personList: Person[],
+  onSelect: (person: Person) => void,
+}
+
+export const PeopleList: React.FC<Props> = React.memo((
+  { personList, onSelect },
+) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {personList.length ? (personList.map(person => (
+          // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid
+          <a
```

Comment:
if a click should cause a redirect, then we use a link tag, otherwise it’s a button tag

Reply:
as i see other pull requests approved with <a> tag, so what should i do? replace it with button tag?